pkg/operator/ceph/cluster/cluster.go

```diff
@@ -572,3 +576,14 @@ func (c *cluster) configureMsgr2() error {
 	}
 	return nil
 }
+
+// set the Rook version in Ceph's k/v store to allow Rook clusters that enable Ceph telemetry to be
+// identified easily as Rook clusters while additionally providing potentially useful information
+// about the version of Rook that is managing the cluster.
+// e.g., rook-version=v1.8.7
+func (c *cluster) setRookVersionInCephKVStore() {
+	ms := config.GetMonStore(c.context, c.ClusterInfo)
+	if err := ms.SetKeyValue(k8sutil.RookVersionLabelKey, rookversion.Version); err != nil {
```

Comment:
If the key already exists from a previous reconcile, it will update the value, right?

Reply:
Right.